tests/unit/amo/sagas/test_users.js

```diff
@@ -203,6 +206,63 @@ describe(__filename, () => {
       expect(calledAction).toEqual(expectedCalledAction);
     });
 
+    it('can receive a non-empty notifications object (dict) in payload', async () => {
+      const state = sagaTester.getState();
+
+      const username = 'babar';
+      const user = createUserAccountResponse({ id: 5001, username });
+
+      const notifications = {
+        reply: false,
+      };
+      const allNotifications = createUserNotificationsResponse();
+      allNotifications[0].enabled = notifications.reply;
+
+      const userFields = {
+        biography: 'I fell into a burning ring of fire.',
+        location: 'Folsom Prison',
+      };
+
+      mockApi
+        .expects('editUserAccount')
```

Comment:
I know this is old code, and I have no idea why I'm just noticing this now, but it seems odd to have a function/action called `editUserAccount`. Shouldn't it be `updateUserAccount`? This is not something that should be addressed in this patch, but I wanted to raise it as a possible code quality issue. What do you think?

Reply:
We never use `updateUserAccount` or `UPDATE_USER_*` for the user profile part. I think that would make sense though, because I introduced some `updateXXX` methods for user notifications for instance.

Reply:
See: https://github.com/mozilla/addons-frontend/issues/5195